packages/data-models/appConfig.ts

```diff
@@ -106,8 +106,16 @@ const APP_ROUTE_DEFAULTS = {
   ],
 };
 
+type IHeaderColourOptions = "primary" | "secondary" | "none";
+
 const APP_HEADER_DEFAULTS = {
   title: "App",
+  collapsing: false,
+  colour: "primary" as IHeaderColourOptions,
+  compact: false,
```

Comment:
nit(non-blocking) could you add a comment to explain what `compact` means. Also is it required or implied by providing heightCompact. If this is more of a style variant would it make more sense to call `variant: 'compact'` (with options for `'default' | 'compact'`)